engine/hid/src/native/hid_gamepad_driver_dinput.cpp

```diff
@@ -490,8 +490,6 @@ namespace dmHID
                     packet.m_Hat[hat_index] = hat_state;
                     hat_index++;
 
-                    assert(button_index == 0);

```

Comment:
It seems we were previously lucky that we had 1 POV, and that it came first in the list.